core/src/components/picker-column/picker-column.scss

```diff
@@ -64,6 +64,41 @@
   scrollbar-width: none;
 }
 
+.picker-item-empty {
```

Comment:
I originally thought the old `.picker-item` CSS wasn't needed. Turns out it's still needed but just for the spacer elements. I updated the selector to make that more clear.